charts/graphscope-store/README.md

```diff
@@ -0,0 +1,116 @@
+# GraphScope Store
+
+[GraphScope Store](https://github.com/alibaba/GraphScope/tree/main/interactive_engine/src/v2/src/main) is a new disk-based row-oriented multi-versioned persistent graph store.
+
+
+## TL;DR
+
+```bash
+$ helm repo add graphscope https://graphscope.oss-cn-beijing.aliyuncs.com/store/charts/
+$ helm install my-release graphscope/graphstore
```

Comment:
could you make the name consistent? maybe graphscope-store? as you describe in the Chart.yml